android/src/main/java/com/reactnativecommunity/webview/events/TopCrashEvent.kt

```diff
@@ -0,0 +1,25 @@
+package com.reactnativecommunity.webview.events
+
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.uimanager.events.Event
+import com.facebook.react.uimanager.events.RCTEventEmitter
+
+/**
+ * Event emitted when the WebView's process has crashed
+ */
+class TopCrashEvent(viewId: Int, private val mEventData: WritableMap) :
```

Comment:
If this is going to be an Android-only event anyway, I would suggest actually using the more-specific name (`RenderProcessGone`) rather than `TopCrash`/`onCrash`. (it could be a bit confusing that the `onCrash` event data contains a boolean variable `didCrash` which may be _false_...)

Reply:
I think there's a subtle difference (which I wasn't sure how to handle). Both cases are crashes (I think), but one was a full crash, and another one just the OS killing the process.  It may be confusing when the webview process is actually done (e.g., the webview being closed), which this event does not fire.  I will take another look and review precisely when these 3 cases/events may happen.

Reply:
What about just renaming `didCrash` to something else, or not including it at all. 

https://developer.android.com/reference/android/webkit/RenderProcessGoneDetail

From the docs, it really seems like the reason is always a crash, but it just indicates if it was observed or not. Thoughts?

Reply:
Nevermind, it makes sense as you said to just rename it to use the same android native event. This way we will be covered for iOS in the future.

Reply:
I think `didCrash` should still be included. I'm not entirely sure of its purpose either (in my experience it's always false), but it doesn't hurt and is one line for you to add while you're here (since you already have it in the docs and types)